document/src/tests/documentupdatetestcase.cpp

```diff
@@ -717,8 +613,8 @@ DocumentUpdateTest::testUpdateArrayEmptyParamValue()
 
     // Verify that the field was set in the document.
     std::unique_ptr<ArrayFieldValue> fval1(doc->getAs<ArrayFieldValue>(field));
-    CPPUNIT_ASSERT(fval1.get());
-    CPPUNIT_ASSERT_EQUAL((size_t) 0, fval1->size());
+    EXPECT_TRUE(fval1.get());
```

Comment:
Consider `ASSERT_TRUE`, is dereferenced in the following expectation

Reply:
Fixed